manifests/07-downloads-deployment.yaml

```diff
@@ -59,7 +59,7 @@ spec:
         - '-c'
         - |
           cat <<EOF >>/tmp/serve.py
-          import BaseHTTPServer, os, re, signal, SimpleHTTPServer, socket, sys, tarfile, tempfile, threading, time, zipfile
+          import BaseHTTPServer, os, re, signal, SimpleHTTPServer, socket, sys, tarfile, tempfile, threading, time, zipfile, errno
```

Comment:
Nit: these imports are currently alphabetized.  Can you slot `errno` in before `os`?

Reply:
I also modified it.